app/src/controllers/chefs.ts

```diff
@@ -6,44 +6,85 @@ import { IdentityProvider } from '../components/constants';
-import type { ChefsFormConfig, ChefsFormConfigData, ChefsSubmissionFormExport } from '../types';
+import type { ChefsFormConfig, ChefsFormConfigData, ChefsSubmissionForm, ChefsSubmissionFormExport } from '../types';
 
 const controller = {
   getFormExport: async (req: Request, res: Response, next: NextFunction) => {
     try {
       const cfg = config.get('server.chefs.forms') as ChefsFormConfig;
-      let formData = new Array<ChefsSubmissionFormExport>();
+      let exportData = new Array<Partial<ChefsSubmissionForm & { formId: string }>>();
 
-      // Get a JSON export of all form data merged into a single array
+      // Get CHEFS export of all form data, converted to a modified ChefsSubmissionForm type
       await Promise.all(
         Object.values<ChefsFormConfigData>(cfg).map(async (x: ChefsFormConfigData) => {
           const data = await chefsService.getFormExport(x.id);
-          data.forEach((d: ChefsSubmissionFormExport) => (d.form.id = x.id));
-          formData = formData.concat(data);
+
+          data.forEach((d: ChefsSubmissionFormExport) => {
+            const financiallySupportedValues = {
+              financiallySupportedBC: isTruthy(d.isBCHousingSupported),
+              financiallySupportedIndigenous: isTruthy(d.isIndigenousHousingProviderSupported),
+              financiallySupportedNonProfit: isTruthy(d.isNonProfitSupported),
+              financiallySupportedHousingCoop: isTruthy(d.isHousingCooperativeSupported)
+            };
+
+            exportData.push({
+              formId: x.id,
+              submissionId: d.form.submissionId,
+              confirmationId: d.form.confirmationId,
+              contactEmail: d.contactEmail,
+              contactPhoneNumber: d.contactPhoneNumber,
+              contactName: `${d.contactFirstName} ${d.contactLastName}`,
+              financiallySupported: Object.values(financiallySupportedValues).includes(true),
+              ...financiallySupportedValues,
+              intakeStatus: d.form.status,
+              latitude: d.latitude,
+              longitude: d.longitude,
+              naturalDisaster: d.naturalDisasterInd,
+              projectName: d.companyNameRegistered,
+              queuePriority: d.queuePriority,
+              singleFamilyUnits: d.singleFamilyUnits ?? d.multiFamilyUnits,
+              streetAddress: d.streetAddress,
+              submittedAt: d.form.createdAt,
+              submittedBy: d.form.username
+            });
+          });
         })
       );
 
+      // Get a list of all submission IDs
+      const result = await chefsService.searchSubmissions({
+        submissionId: exportData.map((x) => x.submissionId as string)
+      });
+
+      // Overwrite export data with application data where possible
+      exportData = exportData.map((x: Partial<ChefsSubmissionForm & { formId: string }>) => {
+        return {
+          ...x,
+          ...result.find((y) => y?.submissionId === x.submissionId)
+        };
+      });
```

Comment:
You can abuse return closures here to avoid needing extra return statements and nesting:
```ts
      exportData = exportData.map((x: Partial<ChefsSubmissionForm & { formId: string }>) => ({
        ...x,
        ...result.find((y) => y?.submissionId === x.submissionId)
      }));
```